user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Style_guide.md

```diff
@@ -0,0 +1,227 @@
+---
```

Comment:
I'm missing some guidelines that were in the ([https://skylinebe-my.sharepoint.com/:w:/g/personal/rens_ampe_skyline_be/EY0HyyoGqONOlIxyDzKAxJgBkSbl0-yP_BrOUsqwMiOnCw?e=3GCFau](word file). I also find some of the guidelines very descriptive. Wouldn't it be better to write each guideline "straight to the point" with an example underneath?

Reply:
what is still missing ?

Reply:
all missing parts have been added after discussion with Rens